core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -189,6 +190,18 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
       );
   }
 
+  putEnglishLanguageNameAtFront(language: string): string {
+    // Check if the language contains the expected format 'abc (something)'.
```

Comment:
One suggestion -- in the docstring comment, change "contains" to "matches".

Reply:
Done.